### DISCUSSION TITLE
Pre-release version command line support

### DISCUSSION THREAD

**Discussion by StoneIron02** [AssetRipper/AssetRipper#1174] [Q&A]
_I'm using a translator._

There was a ParticleSystem-related error in version 0.3.4.0 recently, so I looked up the issue and upgraded it to pre-release version 1.0.4.

However, in the exe file 1.0.4, the command does not work in the command prompt.

Does support exist for this?

**Comment by ds5678** [comment]
No, command line support is now limited to controlling the web API host. However, you can use POST requests to control much larger portions of AssetRipper than was previously possible with command line arguments.

**Reply by StoneIron02** [reply]
OK then, can I get some simple examples of POST requests?

**Reply by ds5678** [reply]
https://github.com/AssetRipper/AssetRipper/blob/4994b282e84b29a26683d43adbd63335cc44b8d4/Source/AssetRipper.GUI.Web/WebApplicationLauncher.cs
